PlottingScripts/Basic_Plot.py

```diff
@@ -0,0 +1,128 @@
+"""This is a basic script for plotting an analysis file."""
+from collections import OrderedDict
+
+import numpy as np
+import pandas as pd
+import pdb
+
+from bokeh.plotting import figure, show, output_file
+
+
+"""This script is just the first step in visualizing the data set. This file
+comprises of the method creating an html bokeh plot and a command at the end to
+implement the method
+"""
+
+
+def makeplot(Dataframe, Cols, minvalues):
+    """Basic method to plot sensitivity anlaysis.
+
+    This is the method to generate a bokeh plot followung the burtin example
+    template at the bokeh website. For clarification, parameters refer to an
+    output being measured (Tmax, C, rxn2, etc.) and stats refer to the 1st or
+    total order sensitivity index.
+
+    Inputs:
+    Dataframe - Name of csv file to be read into as a panda dataframe (String)
+                **Note file must be in same folder as script.**
+    Cols- Names of the columns to be plotted (Array of strings)
+    minvalues- Cutoff minimum for which parameters should be plotted (Array of
+                floats)
+    Outputs:
+    Generates an html of the plot. The axis is a log scale.
+    """
+    # Read in csv file as panda dataframe.
+    tdf = pd.read_csv(Dataframe, delimiter=' ', skipinitialspace=True,
```

Comment:
The Dataframe parameter of your function should be updated to simply **be** the dataframe that you define as tdf.  I have a function (`get_sa()`) that will read all the analysis files and clean/build the pandas dataframes for all output measures.  That function will provide the cleaned dataframe to use as input for your makeplot.  As far as specific changes to your code here this would mean deleting line 35 and just changing line 36 to `df = Dataframe` (and updating the function documentation above accordingly).